client/src/components/Collections/FixedIdentifierDatasetCollectionElementView.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import type { HDASummary } from "@/api";
+import localize from "@/utils/localization";
+
+interface Props {
+    element: HDASummary | undefined;
+    hasActions?: boolean;
+}
+
+defineProps<Props>();
+
+const emit = defineEmits<{
+    (event: "element-is-discarded", element: any): void;
```

Comment:
If we trust the prop type:
```suggestion
    (event: "element-is-discarded", element: HDASummary): void;
```